productivity.md

```diff
@@ -0,0 +1,75 @@
+
+- Talk to the product team early and often. Understanding the requirements is key to planning work accurately.
+- Collaborate with the product team if you have ideas for how to improve and/or simplify the work. They are aligned to our goal of delivering high-quality work quickly, so suggestions are welcomed.
+- Some work is very difficult to plan by just looking at it. Make prototypes to help gain a better understanding and identify potential issues.
+- Ask: What are some non-obvious interactions that might cause problems?
+- In planning meetings, get everyone to speak, so everyone is able to share their unique perspective. There might be long periods of silence while folks are thinking, and that’s ok.
+
+### Handling grossly misestimated tickets
+
+- Work with the backlog owner to reduce the scope and update the DoD.
+- Create additional tickets to encompass the remaining work.
+- Working with the backlog owner, determine whether the additional tickets should be pulled into the current sprint or prioritized for an upcoming sprint.
+
+## Troubleshooting individual productivity
+
+### Consistently low velocity (typically junior)
+
+Low velocity is usually a symptom of either time management issues or taking tickets that are too difficult.
+
+- Limit work-in-progress. You should never have more than 3 tickets open at a time. Ideally you should only ever 1 in progress and 1-2 in PR. Focus on one thing at a time to reduce the need for context switching.
+- Use the guide for [prioritizing work](#prioritizing-work) to decide what you should be working on.
+- Take small, well-defined tickets. Avoid tickets 3-points or higher, as these usually contain large areas of uncertainty.
+- Before starting any ticket, make a plan for how you will accomplish the ticket. Present that plan to a more senior engineer and get feedback.
+  - For larger tasks, repeat this process daily, revising your plan as you go along.
+- If you struggle with rabbit-holing, set a repeating timer for 30 minutes. When the timer goes off, if you don’t feel like you’re making meaningful progress, reach out for help.
+  - Once you are consistently making progress, increase the timer interval until you feel confident that you can notice when you’re stuck and need help.
+
+As a manager of a team member struggling with low velocity, begin by asking open-ended questions to get the developer to reflect on why they’ve been struggling to deliver tickets.
+
+- How does the person feel about their tickets?
+  - Can they identify any trends in the problems they encounter with tickets?
+- How do they feel asking for help?
+  - What prevents them from noticing that they should ask for help?
+  - What prevents them from reaching out for help when they notice they’re stuck?
+
+### Unpredictable / highly variable velocity (typically mid-level)
+
+When a developer’s productivity fluctuates dramatically, this is often a symptom of taking on highly complex work relative to their level.
```

Comment:
While I agree, I'm not sure that this is a sufficient explanation. 

Mid-level Engineers can still struggle with estimation, those lessons also apply in this section.

Reply:
In my mind, estimation is a team a team-level problem, not an individual problem. Tho, I do think we can expand on this. Aside from "taking on highly complex work relative to their level," what do you think EN2s struggle with during the transition between EN1 and EN3?

Reply:
In terms of team-level vs. individual problem, I think it's both. At the end of the day, the Engineer taking on the ticket has to take responsibility for the estimate -- if Engineers don't ask "what's the right estimate of this ticket _for me_?" that can lead to estimation issues. 

To your point around what EN2s struggle with, highly variable velocity tells me some combination of the following are true: 

- Tickets are being estimated poorly. 
- Our time is going elsewhere besides the tickets initially our sprint (late-adds, meetings, etc.)
- Complexity is too high

You touch on the complexity piece here and we covered estimation, I just might add a note on the second bullet.